python_binding/lsm/_data.py

```diff
@@ -357,12 +357,26 @@ class System(IData):
     STATUS_PREDICTIVE_FAILURE = 1 << 4
     STATUS_OTHER = 1 << 5
 
-    def __init__(self, _id, _name, _status, _status_info, _plugin_data=None):
+    def __init__(self, _id, _name, _status, _status_info, _plugin_data=None,
+                 _fw_version=''):
```

Comment:
As we need to raise `NO_SUPPORT` error when fw_version is empty, we cannot
depend on `@default_property` decoration.